moveos/moveos-stdlib/src/natives/mod.rs

```diff
@@ -82,6 +84,7 @@ pub fn all_natives(gas_params: GasParameters) -> NativeFunctionTable {
     );
     add_natives!("rlp", moveos_stdlib::rlp::make_all(gas_params.rlp));
     add_natives!("bcd", moveos_stdlib::bcd::make_all(gas_params.bcd));
+    add_natives!("emit", moveos_stdlib::events::make_all(gas_params.emit));
```

Comment:
This is the module name, it should be `events`. The `emit` function name is defined in the `events.rs`